modules/setting/indexer.go

```diff
@@ -71,6 +73,7 @@ func loadIndexerFrom(rootCfg ConfigProvider) {
 	Indexer.IssueIndexerName = sec.Key("ISSUE_INDEXER_NAME").MustString(Indexer.IssueIndexerName)
 
 	Indexer.RepoIndexerEnabled = sec.Key("REPO_INDEXER_ENABLED").MustBool(false)
+	Indexer.RepoIndexerUnits = strings.Split(sec.Key("REPO_INDEXER_REPO_TYPES").MustString("sources,forks,mirrors,templates"), ",")
```

Comment:
If I remember correctly, there is a function called `Strings` or something, which can trim spaces for every field?

Reply:
Yes, there is, but it doesn't work with "MustString" so you can't set a default with it.

Reply:
The ini package is really problematic .... 😭